src/main/java/com/std/tothebook/api/entity/MyBook.java

```diff
@@ -0,0 +1,71 @@
+package com.std.tothebook.api.entity;
+
+import com.std.tothebook.api.enums.MyBookStatus;
+import lombok.Getter;
+import javax.persistence.Entity;
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Table(name = "my_book")
+public class MyBook {
+
+    // id
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    // 고객 id
+    @ManyToOne
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```
fetch type을 지정해주지 않으면 EAGER로 설정됩니다.
이 경우 조회 시 join이 되어 조회가 될 텐데 이 경우를 고려하신 걸까요?
만약 그게 아니라면 즉시로딩과 지연로딩에 관해 찾아보시는 걸 추천 드립니다!

Reply:
fetch = FetchType.LAZY와 EAGER에 대하여 이번에 알게 되었습니다.
EAGER를 사용하면 성능 면에서 리스크를 안고 가기 때문에 선호하지 않는군요.😄 피드백 감사합니다.